pkg/kubelet/server/server.go

```diff
@@ -358,6 +364,9 @@ func (s *Server) InstallDefaultHandlers() {
 		healthz.NamedCheck("syncloop", s.syncLoopHealthCheck),
 	)
 
+	if utilfeature.DefaultFeatureGate.Enabled(metricsfeatures.ComponentSLIs) {
+		slis.SLIMetricsWithReset{}.Install(s.restfulCont)
```

Comment:
Does this have any interactions with line 387?
```
	s.restfulCont.Handle(metricsPath, legacyregistry.Handler())
```

Does the ordering of these lines matter?

Reply:
No, ordering between those lines do not matter.